plugins/polio/migrations/0167_migrate_budget_data.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.9 on 2024-02-07 16:28
+
```

Comment:
I doubt that Django generated that migration or it's way smarter than I thought. 

Joke aside, this migration is what I expected to see indeed.